ci/setup-gdk.ps1

```diff
@@ -1,129 +1,18 @@
-        "package", `
-        "retrieve", `
-        "worker_sdk", `
-        "c-dynamic-x86_64-gcc510-linux", `
-        "$($pinned_core_sdk_version)", `
-        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc510-linux.zip" `
-    )
-    Finish-Event "download-worker-sdk-x86_64-gcc510-linux" "download-spatial-packages"
-
-    Start-Event "download-worker-sdk-csharp" "download-spatial-packages"
-    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
-        "package", `
-        "retrieve", `
-        "worker_sdk", `
-        "csharp", `
-        "$($pinned_core_sdk_version)", `
-        "$($core_sdk_dir)\worker_sdk\csharp.zip" `
-    )
-    Finish-Event "download-worker-sdk-csharp" "download-spatial-packages"
-    Finish-Event "download-spatial-packages" "setup-gdk"
-
-    Start-Event "extract-spatial-packages" "setup-gdk"
-    Expand-Archive -Path "$($core_sdk_dir)\tools\schema_compiler-x86_64-win32.zip" -DestinationPath "$($binaries_dir)\Programs\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\schema\standard_library.zip" -DestinationPath "$($binaries_dir)\Programs\schema\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c_headers.zip" -DestinationPath "$($binaries_dir)\Headers\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86-vc140_md-win32.zip" -DestinationPath "$($binaries_dir)\Win32\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-vc140_md-win32.zip" -DestinationPath "$($binaries_dir)\Win64\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc510-linux.zip" -DestinationPath "$($binaries_dir)\Linux\" -Force
-    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\csharp.zip" -DestinationPath "$($binaries_dir)\Programs\worker_sdk\csharp\" -Force
-    Finish-Event "extract-spatial-packages" "setup-gdk"
-
-    # Copy from binaries_dir
-    Copy-Item "$($binaries_dir)\Headers\include\*" "$($worker_sdk_dir)\" -Force -Recurse
-
-    Start-Event "build-utilities" "setup-gdk"
-    $msbuild_proc = Start-Process -PassThru -NoNewWindow -FilePath "$($msbuild_exe)" -ArgumentList @(`
-        "/nologo", `
-        "SpatialGDK\Build\Programs\Improbable.Unreal.Scripts\Improbable.Unreal.Scripts.sln", `
-        "/property:Configuration=Release",`
-        "/restore" `
-    )
-
-    # Note: holding on to a handle solves an intermittent issue when waiting on the process id
-	$msbuild_handle = $msbuild_proc.Handle
-    Wait-Process -Id (Get-Process -InputObject $msbuild_proc).id
-    if ($msbuild_proc.ExitCode -ne 0) {
-        Write-Log "Failed to build utilities. Error: $($msbuild_proc.ExitCode)"
-        Throw "Failed to build utilities."
+pushd $gdk_in_engine
+    if (-Not (Test-Path env:NO_PAUSE)) { # seems like this is set somewhere previously in CI, but just to make sure
+        $env:NO_PAUSE = 1
     }
-    Finish-Event "build-utilities" "setup-gdk"
+    $env:MSBUILD_EXE = $msbuild_path
```

Comment:
Do we still need to do this now that the Setup.bat finds the VS install on the CI agent?

Reply:
I don't think it finds it, since we only have the build tools, not the full visual studio, but I will test it out.